ruby-gem/bin/configure

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+lib_path = File.expand_path('../lib', __dir__)
+$:.unshift(lib_path) if !$:.include?(lib_path)
+require 'configure'
+
+def help
+	puts <<~HELP
+		configure
+		A command-line utility for applying configuration secrets with strong encryption
+
+		USAGE:
+		bundle exec configure <SUBCOMMAND>
+
+		SUBCOMMANDS:
+		init          Change mobile secrets settings
+		apply         Decrypt the current mobile secrets for this project
+		update        Update this project's encrypted mobile secrets to the latest version
+		help          Prints this message
+
+	HELP
+end
+
+case ARGV.first
+when 'init'
+  Configure.init()
```

Comment:
YIL (_Yesterday_ I Learned) that RuboCop recommends not using `()` for method called with no arguments.

🤷‍♂️ 

```suggestion
  Configure.init
```